lib/glimesh_web/templates/user_settings/stream.html.eex

```diff
@@ -1 +1,52 @@
-<h2>If you could stream, your settings would go here.</h2>
+<div class="container">
+    <h2 class="mt-4"><%= gettext("Stream Settings") %></h2>
+    <%= if !@channel do %>
+        <h5><%= gettext("You currently don't have a channel setup. Press the button below if you'd like to setup your channel!") %></h5>
+        <%= button gettext("Create Channel"), class: "btn btn-primary", to: Routes.user_settings_path(@conn, :create_channel), method: "put"%>
```

Comment:
We can do it in another PR, but we might want to just consider making this the create channel page if they don't have one already.

Reply:
Let's do it in a different PR. That way it doesn't delay this one any more than it already is since I would wanna spend some decent time on making that page pretty. :)